sdk/nodejs/appservice/zMixins.ts

```diff
@@ -750,8 +779,10 @@ export function combineAppSettings(settings: pulumi.Input<{[key: string]: string
 }
 
 /** @internal */
-export function combineBindingSettings(trigger: appservice.BindingSettings, inputOutputs?: appservice.BindingSettings[]) {
-    const all = [trigger, ...inputOutputs || []];
+export function combineBindingSettings(trigger: BindingSettings<BindingDefinition>,
+                                       inputs?: InputBindingSettings[],
+                                       outputs?: OutputBindingSettings[]) {
+    const all = [trigger, ...inputs || [], ...outputs || []];

```

Comment:
oh wow.   i didn't know that works.  looks like that parses as `...(inputs || [])`.